.github/workflows/artifacts.yml

```diff
@@ -109,7 +109,7 @@ jobs:
   sentry:
     name: Send build info to sentry
     if: >
-      github.event.workflow_run.event == 'pull_request' &&
+      github.event.workflow_run.event == 'push' &&
       github.event.workflow_run.conclusion == 'success'
```

Comment:
Since we can technically release a failied build (let's say it didn't succeed on tests), I think we should also remove `conlcusion` condition.

Reply:
Fixed